src/utils/output.js

```diff
@@ -17,17 +17,24 @@ function prettify(caption, deps) {
   return list.length ? [caption].concat(list) : [];
 }
 
-export default function output(result, log, json) {
+export default function output(result, log, argv) {
   return new Promise(resolve => {
-    if (json) {
-      log(JSON.stringify(result, replacer));
+    if (argv.json) {
+      const clone = Object.assign({}, result);
+      if (!argv.used) delete clone.used;
+      log(JSON.stringify(clone, replacer));
     } else if (noIssue(result)) {
-      log('No depcheck issue');
+      const unused = ['No depcheck issue'];
```

Comment:
I am not sure if we need to show ***Used dependencies* in this case. Maybe user with the common scenario are not going to see a long list. However, if you are going to show the list, I think it could better to remove the **No depcheck issue** caption.